src/app/space/create/deployments/apps/collapsible-deployment-info.component.html

```diff
@@ -0,0 +1,22 @@
+<div [collapse]="collapsed">
+  <hr>
+  <deployments-donut [applicationId]="applicationId" [environment]="environment" [spaceId]="spaceId" [mini]="false"></deployments-donut>
+
+  <div class="deployment-chart">
+    <div class="chart-column">
+      <pfng-chart-sparkline class="chart-sparkline" [config]="cpuConfig" [chartData]="cpuData"></pfng-chart-sparkline>
+    </div>
+    <div class="chart-column">
+      <deployment-graph-label type="CPU" dataMeasure="Cores" [value]="cpuVal" [valueUpperBound]="cpuMax"></deployment-graph-label>
+    </div>
+  </div>
+
+  <div class="deployment-chart">
+    <div class="chart-column">
+      <pfng-chart-sparkline class="chart-sparkline" [config]="memConfig" [chartData]="memData"></pfng-chart-sparkline>
+    </div>
+    <div class="chart-column chart-text">
```

Comment:
While you're here, can you figure out why one of these graph container divs has `class="chart-column chart-text"` and the other only has `class="chart-column"` and make them match?